### ISSUE TITLE
BooleanUserOverrideGather - Add a "Excluding headless" bool

### ISSUE THREAD

**Issue by 3x1t-5tyl3** [Yellow-Dog-Man/Resonite-Issues#3132]
### Is your feature request related to a problem? Please describe.

Basically when running headless' this is a great component for when you wanna do some fun stuff.
Sadly there's no way of using this component without Flux to make sure the headless is ignored.

### Describe the solution you'd like

Another bool to allow me to exclude headless' from the component or more fields to exclude the headless.
So I can more easily check against the headless to make life easier.

### Describe alternatives you've considered

In Flux always add a override for the headless.

### Additional Context

This seems like an underused component from what I can tell by the userbase. And I just now learned about it.
Sadky I wanted to immedeatly use it for "Halloween shenanigans" where if every user has the local enabled it would do something spooky (maybe™

### Requesters

3x1t_5tyl3

**Comment by JackTheFoxOtter** [comment]
The component is pretty fantastic! But I'm a bit confused, because if you add overrides on write, then they will only be added for the headless if it writes to the field. Which isn't really a problem if you just make it not write to the field in the first place. Even if you use button interactions and such to write to the value, it will never have a value for the headless in the first place, since it can't trigger those.

**Comment by 3x1t-5tyl3** [comment]
> The component is pretty fantastic! But I'm a bit confused, because if you add overrides on write, then they will only be added for the headless if it writes to the field. Which isn't really a problem if you just make it not write to the field in the first place. Even if you use button interactions and such to write to the value, it will never have a value for the headless in the first place, since it can't trigger those.

I guess I could've worded it bettter: Let me try again.

Basically on a headless these values are always *biased* towards a user count of +1 on a headless.
As the headless doesn't interact; It just hosts. As such the "All" check for this specific component requires me to either have a persistant override for the headless (which I do not want to do), or requires me to add an extra bool check in flux to test against it.

Which IMO goes against the whole concept. I should be able to determine all *users* not *all accounts* in the world.
I don't see the headless as a user. ***Eventho it technicalyl is I am aware***

**Comment by JackTheFoxOtter** [comment]
Ah okay, right that's a thing I didn't consider. Yeah that would be a useful addition then!

**Comment by Frooxius** [comment]
Added in 2024.10.29.1229, thanks!
